bin/app-server.sh

```diff
@@ -21,29 +21,12 @@
 # - ZOWE_ZLUX_TELNET_PORT
 # - ZOWE_ZLUX_SECURITY_TYPE
 
-if [ -n "$NODE_HOME" ]
+if [ -z "${ROOT_DIR}" ]
 then
-  NODE_BIN=${NODE_HOME}/bin/node
-  export PATH=${NODE_HOME}/bin:$PATH
-elif [ -n "$ZOWE_NODE_HOME" ]
-then
-  NODE_BIN=${ZOWE_NODE_HOME}/bin/node
-  export PATH=${ZOWE_NODE_HOME}/bin:$PATH
-else
-  NODE_BIN=node
-fi
-export _BPXK_AUTOCVT=ON
-
-nodeVersion="$(${NODE_BIN} --version)"
-nodeMajorVersion=$(echo ${nodeVersion} | cut -c2-3)
-if [ $nodeMajorVersion = "12" ]
-then
```

Comment:
this stuff is not only z/os specific but is handled globally so it's probably not necessary to do in our code, but for devs it would still be needed